Test/PerformanceCollector/FunctionalTests/CommonTests.cs

```diff
@@ -183,8 +183,25 @@ internal static void QuickPulseTopCpuProcesses(QuickPulseHttpListenerObservable
             Assert.IsTrue(samples.Count > 0);
 
             Assert.IsTrue(samples.Any(s => s.TopCpuProcesses.Length > 0));
-            Assert.IsFalse(
-                samples.Any(s => s.TopCpuProcesses.Any(p => string.IsNullOrWhiteSpace(p.ProcessName) || p.CpuPercentage < 0 || p.CpuPercentage > 100)));
+
+            try
+            {
+                Assert.IsFalse(
+                    samples.Any(s => s.TopCpuProcesses.Any(p => string.IsNullOrWhiteSpace(p.ProcessName) || p.CpuPercentage < 0 || p.CpuPercentage > 100)));
+            }
+            catch
+            {
+                var weirdSample = samples.First(s => s.TopCpuProcesses.Any(p => string.IsNullOrWhiteSpace(p.ProcessName) || p.CpuPercentage < 0 || p.CpuPercentage > 100));
+
+                Console.WriteLine("Top CPU test failed, weird sample found. Processes:");
```

Comment:
Does console write works? Will this show up in build logs? I think Trace.Write. is what other tests use.